libraries/typings-generator/src/TypingsGenerator.ts

```diff
@@ -152,6 +185,21 @@ export class TypingsGenerator {
     }
   }
 
+  private _clearDependencies(target: string): void {
+    const dependencies: IterableIterator<string> | undefined = this._targetMap.get(target)?.keys();
```

Comment:
Split this out into two lines so that you can avoid the `delete()` call if you know the map doesn't exist

Reply:
Not sure this is an issue anymore. Can you check the latest update?

Reply:
Actually, since it doesn't seem that you ever use the value in the contained map (the bool), why not use `Set<string>`? This would simplify querying and removal of these values

Reply:
Converted to a Set. Looks better IMO.